runners/katacoda/index.ts

```diff
@@ -209,6 +209,21 @@ export class Katacoda extends Runner {
 
     }
 
+    runCloneRepository(step: Step, command: Command): RunResult {
+
+        let cdCommand = this.changeCurrentDir(path.join("/root", "devonfw", "workspaces", "main"));
+        let workspaceDir = path.join("devonfw", "workspaces", "main");
+        let directoryPath = path.join(workspaceDir, command.parameters[0]).replace(/\\/g, "/");
```

Comment:
We are not going to use devon ide in every tutorial. The command has to work without ide, as well. So better use the relative path to the root-directory.

Reply:
-Do we have to change createFile and changeFile, too? They are also written to be solely used in workspaceDir.
-That's right, then we have to change this too. I think the command should work even without a Devon Ide. @maybeec what do you think?